fetchai/ledger/serialisation/transaction.py

```diff
@@ -120,7 +118,6 @@ def encode_payload(buffer: io.BytesIO, payload: transaction.Transaction):
                 buffer.write(shard_mask_bytes)
 
         if SMART_CONTRACT == contract_mode or SYNERGETIC == contract_mode:
-            address.encode(buffer, payload.contract_digest)
```

Comment:
We should increment the transaction serialisation version (line 13 of this file)